config/networkpolicy/monitoring/monitoring.yaml

```diff
@@ -0,0 +1,48 @@
+# Defines ingress rules for specific port. These ports are defined by
+# the services residing in redhat-ods-monitoring. namespaceSelector
+# ensures that traffic from only the desired namespaces is allowed
+# 9114 for blackbox or user_facing_endpoints* all down
+# 9115 for blackbox health
+# 10443 and 9091 for web
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: redhat-ods-monitoring
+  namespace: redhat-ods-monitoring
+spec:
+  podSelector: {}
+  ingress:
+    - ports:
+        - protocol: TCP
+          port: 443
```

Comment:
Is this all the ports that some component possible could have something to monitor on? A set of standardized ports could have some value, or possible open it up to be defined by the component it self.

Reply:
these are the ones copied over from what we are using now, not refined anything in the draft yet.
but i think the goal is to be able to migrate function first. we can do the change independently,